lap-neo4j-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Administrator user name used when provisioning virtual machines"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Administrator password used when provisioning virtual machines"
+      }
+    },
+    "dnsLabelPrefix": {
+   "type": "string",
+   "metadata": {
+      "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+    }
+    },
+    "lapVmSize": {
+      "type": "string",
+      "defaultValue": "D1_V2",
```

Comment:
@brijrajsingh the size still has the Standard word in it :) SHould be Standard_D1_v2